designer/server/src/test/scala/pl/touk/nussknacker/ui/api/ManagementResourcesSpec.scala

```diff
@@ -422,7 +422,9 @@ class ManagementResourcesSpec
       CustomActionRequest(ScenarioActionName("non-existing"))
     ) ~> check {
       status shouldBe StatusCodes.NotFound
-      responseAs[String] shouldBe "non-existing is not existing"
+      responseAs[
+        String
+      ] shouldBe s"Couldn't find definition of action non-existing for scenario ${ProcessTestData.sampleProcessName} when trying to validate"
```

Comment:
Maybe "Couldn't find definition of action non-existing" is sufficient?

Reply:
I think it is good to write the scenario name here to make sure that we are actually processing what we want. More information shouldn't hurt :)

Reply:
Ok, lets keep the scenario name. The part "when trying to validate" is not true. With current implementation it's more like "when trying to execute an action" and is obsolete.